File: JAVA/Kata/Kata.java
public class Kata{

	public static boolean isEven(int number){

	if (number % 2 == 0){
	return true;
	}else{
	return false;
	}
	}

	public static boolean isPrime(int number){

	if (number == 1 || number == 3 || number == 5 || number == 7 || number == 11 || number == 13){

		return true;
	}else if (number % 1 == 0 || number % 3 == 0 || number % 5 == 0 || number % 7 == 0 || number % 11 == 0 || number % 13 == 0){
		return false;
	}else{
		return true;
	}

	}
	
	public static long factorialOf(int number){
	long total = 1;
	while(number > 0){
	
	total = total*number;
	number--;
	}
	return total;
	}

	public static boolean isPalindrome(int number){
	
	if(number < 0){
	return false;
	}
	int divisor = 0;
	int toStore = 0;
	while(number != 0){
	divisor = number%10;
	toStore = toStore*10+divisor;
	number/=10;
	}
	return true;
	}

	public static long squareOf(int number){
	
	return (long) Math.pow(number, 2);
	}
	
	public static int subtract(int number1, int number2){
	
	int difference = number1-number2;
	
	if (difference > 0){
	return difference;
	}else{
	return difference*(-1);
	}
	}

	public static int factorOf(int number){

	int count = 0;
	int nobetheDivisor = 0;
	while(nobetheDivisor <= number){
	nobetheDivisor++;
	if(number % nobetheDivisor == 0){
	count++;
	}
	}
	return count;
	}

	public static boolean isSquare(int number){
	
	if(number%Math.sqrt(number)==0){
	return true;
	}else{
	return false;
	}
	
	}

	public static float divide(int number1, int number2){

	int count = 0;
	if(number2==0){
	return 0;
	}
	int result = 0;
	result = number1/number2;
	return result;
	}
}

File: PYTHON/Kata/kata.py
"""
def is_even(number):
	if number %2 == 0:
		return True
	else:
		return False
print(is_even(24))

def is_prime(number):
	if number == 1 or number == 3 or number == 5 or number == 7 or number == 11 or number == 13:
		return True
	elif number % 1== 0 or number % 3== 0 or number % 5 == 0 or number % 7==0 or number % 11==0 or number %13 ==0:
		return False
	else:
		return True
print(is_prime(18))

def subtract(number1, number2):
	difference = number1-number2
	if difference > 0:
		return difference
	else:
		return difference*(-1)
print(subtract(12, 15))

def divide(number1, number2):
	if number2 == 0:
		return 0
	else:
		quotient = number1/number2
		return quotient
print(divide(12, 0))

def factor_of(number):
	divisor = 0
	the_number = 0
	while(the_number<=number):
		the_number+=1
		if number % the_number:
			return divisor++1
			
print(factor_of(10))

import math

def is_square(number):
	if number%math.sqrt(number)==0:
		return True
	else:
		return false

print(is_square(25))

"""

def is_palindrome(number):
	if number <0:
		return False
	number1 =0
	number2 =0
	while number !=0 :
		if number==5:
			number1= number%10
			number2= number2*10+number1
			number/=10
			return True
		
print(is_palindrome(1234))

def factorialOf(number):
	sum = 1
	while number != 0:
		sum*=number
		number-=1
	return sum
print(factorialOf(2))

def squareOf(number):
	to_calculate= math.pow(number, 2)
	return to_calculate
print (squareOf(3))







